pytest_fixtures/core/sys.py

```diff
@@ -10,7 +10,7 @@
 def foreman_service_teardown(target_sat):
     """stop and restart of foreman service"""
     yield target_sat
-    target_sat.execute('satellite-maintain service start --only=foreman')
+    target_sat.execute('satellite-maintain service start')
```

Comment:
you might want to update this step here as well, https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/destructive/test_contenthost.py#L74

It would be best to remove this fixture entirely and add the `request.addfinalizer` call in the test itself for teardown, after replacing this last usage of this fixture with target_sat here, https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/destructive/test_foreman_service.py#L23

Reply:
I think having the fixture is fine, so that its easier to use and we're not repeating the call and lambda definition wherever its needed.

Also allows for easier growth of the service teardown cleanup that can be applied broadly. If we want to add a service status assert here too? easy - without updating every use.